client/src/components/BottomPanel/components/ChromosomeMap/ChromosomeMap.tsx

```diff
@@ -1,19 +1,19 @@
-import React, { useMemo } from "react";
+import React, { useMemo, useRef, useEffect } from "react";
 import { useCoverageQuery } from "common/queries/coverage";
 import { SKELETON } from "@blueprintjs/core/lib/esnext/common/classes";
 import { useSelector } from "react-redux";
 import { RootState } from "reducers";
-// import { useChromatinViewerSelectedGene } from "common/queries/useChromatinViewerSelectedGene";
+import { useChromatinViewerSelectedGene } from "common/queries/useChromatinViewerSelectedGene";
 import { ScaleBar } from "../ScaleBar/ScaleBar";
 import { CoveragePlot } from "../CoveragePlot/CoveragePlot";
 import { GeneMap } from "../GeneMap/GeneMap";
 import { Cytoband } from "../Cytoband/Cytoband";
 import { CoverageAtScale } from "./style";
 
 export const ChromosomeMap = () => {
-  const chromosome = "chr2";
-  const BAR_WIDTH = 8;
-  // const { selectedGene } = useChromatinViewerSelectedGene();
+  const BAR_WIDTH = 6;
```

Comment:
nit: Can we add some comment around how 6 was determined please 😄 🙏 ? Thank you!